tests/test_basic.py

```diff
@@ -439,12 +446,12 @@ def test_size_and_aligment(cs: cstruct) -> None:
 
 def test_dynamic_substruct_size(cs: cstruct) -> None:
     cdef = """
-    struct {
+    typedef struct {
```

Comment:
I'm not a C expert but I think prior to this change, `sub` was the variable name and not a structure name, so this is now technically more correct? But is it "nicer"? Or should we still allow this kind of definition, if not just for the sake of backwards compatibility?

Reply:
I made a suggested fix for this with https://github.com/fox-it/dissect.cstruct/commit/83505dd40fce387ade688b73549b9735360ec6d9

Reply:
I'll have a look over once more.

Reply:
Probably good 👍